apollo-common/src/main/resources/application.properties

```diff
@@ -17,3 +17,5 @@ server.max-http-header-size=10240
 management.endpoints.web.base-path=/
 logging.file.max-size=50MB
 logging.file.max-history=10
+
+management.endpoints.web.exposure.include=*
```

Comment:
It is dangerous to expose all the endpoints, the default is `info, health`, so how about change it to `info,health, prometheus `